react/components/organisms/ripe-image/ripe-image.js

```diff
@@ -250,7 +250,11 @@ export class RipeImage extends mix(Component).with(LogicMixin) {
             /**
              * Style to be applied to the image, used for zoom application.
              */
-            style: PropTypes.object
+            style: PropTypes.object,
+            /**
+             * Callback called when an error occurs while loading the image.
+             */
+            onError: PropTypes.func

```

Comment:
Isn't `onLoad` missing?

Reply:
Already on the logic mixin, since its used by all components: https://github.com/ripe-tech/ripe-sdk-components-react/blob/e9efe5a8e586783e678adf16a9ed17325dc2e8a2/react/mixins/logic.js#L107

Reply:
What I mean is a function that is called when the image actually loads, as you did in Vue. That one from the mixin seems like dead code.

Reply:
Its called here: https://github.com/ripe-tech/ripe-sdk-components-react/blob/ac27f951aea368ee89cf716d5c0f8c680628c32a/react/components/organisms/ripe-image/ripe-image.js#L595
The function is "declared" inside the logic mixin but all components receive that function as prop and use it inside the component, not in the mixin. Its to replicate the same organization that exists in the vue components.